src/solver/conv_hip_implicit_gemm_fwd_xdlops.cpp

```diff
@@ -227,6 +233,50 @@ void RunCKSolution(const Handle& handle,
     }
 }
 
+template <typename DataType>
+InvokerFactory MakeInvokerFactoryHelper(CKArgs ck_args, size_t config_idx)
+{
+    return std::move([ck_args = std::move(ck_args),
+                      config_idx](const std::vector<Kernel>& kernels) mutable {
+        std::ignore = kernels;
+
+        return std::move([ck_args = std::move(ck_args), config_idx](
+                             const Handle& handle, const AnyInvokeParams& primitive_parameters) {
+            RunCKSolution<DataType>(handle, primitive_parameters, ck_args, config_idx);
+        });
+    });
```

Comment:
`moving` something returned by value disables RVO.
But as far as it's supposed to be called only once, it shouldn't be a problem.

Reply:
Good catch. That `move` needs to be removed. I was trying other stuff with trying to make moving work with `ck`'s `unique_ptrs`.

Reply:
Implemented in f4d0831